app/controllers/declaration/ItemsSummaryController.scala

```diff
@@ -43,35 +43,26 @@ class ItemsSummaryController @Inject()(
 
   def addItem(): Action[AnyContent] = (authenticate andThen journeyType).async { implicit request =>
     val newItem = ExportItem(id = exportItemIdGeneratorService.generateItemId())
-    updateCache(journeySessionId, newItem)
+    exportsCacheService
+      .update(
```

Comment:
This is implemented in `ModelCacheable` - 

Maybe moving adding logic to `ExportsDeclaration` would nice close this object. It only depends on itself. I am thinking about something like:
```
_.addItem()
```